src/components/AssetsComponent.jsx

```diff
@@ -128,7 +138,7 @@ AssetsComponent.propTypes = {
 AssetsComponent.defaultProps = {
   assets: [],
   assetsCount: 0,
-  getAssetsAction: () => {}
+  getAssetsAction: () => { }
```

Comment:
I believe this should be `getAssetsAction: () => {}`

Reply:
done.